.github/workflows/playwright.yml

```
@@ -29,12 +29,14 @@ jobs:
       - name: Install Playwright Browsers
         run: npx playwright install --with-deps
       - name: Set up environment and run tests
+        env:
+            NEXT_PUBLIC_HUBSPOT_PORTAL_ID: ${{ secrets.NEXT_PUBLIC_HUBSPOT_PORTAL_ID }}
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 10 but found 12

(indentation)